aidants_connect_carto_api/migrations/0003_auto_20200415_1800.py

```diff
@@ -0,0 +1,174 @@
+# Generated by Django 3.0.5 on 2020-04-15 16:00
```

Comment:
Petite suggestion d'amélioration qualité : après un `makemigrations`, ça peut être cool de renommer le fichier généré par Django en quelque chose de plus sémantique :)

C'est un coup à prendre, ça m'arrive aussi d'oublier, mais je pense que ça a de la valeur ^^

[Cet article](https://adamj.eu/tech/2020/02/24/how-to-disallow-auto-named-django-migrations/) rentre plus en détail dans le sujet, et propose aussi quelques pistes concrètes pour améliorer ça !

/ping @Anna-Livia au passage, je pense que l'article peut aussi t'intéresser ^^

Reply:
en effet je me suis toujours demandé pourquoi parfois django renommé le fichier, et parfois non :) je vais le faire. ou alors je migre tout dans les deux premiers 🙃

Reply:
Je crois que la capacité de Django à le faire tout seul dépend principalement de la quantité de changements dans la migration générée. S'il y en a trop cela devient impossible pour lui de donner un nom pertinent, et il se rabat sur du `..._auto_...` :)

Reply:
j'ai réintégré tout dans les 2 premières migrations :)